src/main/java/org/cyclonedx/parsers/JsonParser.java

```diff
@@ -50,8 +48,9 @@
      */
     public Bom parse(final File file) throws ParseException {
         try {
-            final String jsonString = IOUtils.toString(new FileInputStream(file), StandardCharsets.UTF_8);
-            return JSON.parseObject(jsonString, Bom.class, Feature.SupportNonPublicField);
+            ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
you could keep this as a private attribute (in the class level) instead of instantiating in each method call

Reply:
Addressed in: adfcf52807055bd55f40406e8c0c9a5cfa468f9e